src/tqec/detectors/operation.py

```diff
@@ -31,7 +31,8 @@ def __init__(self, *shifts: int) -> None:
         with the correct tags. Otherwise, you might need to manually tag the operation with the
         `cirq.VirtualTag` and the `STIM_TAG`.
 
-        :param shifts: How much to shift each coordinate.
+        Args:
+            shifts: How much to shift each coordinate.
```

Comment:
```suggestion
            *shifts: How much to shift each coordinate.
```
I think Google style also includes the `*`.